snapshot/snapshotter.go

```diff
@@ -296,6 +296,15 @@ type Snapshotter interface {
 	// Walk all snapshots in the snapshotter. For each snapshot in the
 	// snapshotter, the function will be called.
 	Walk(ctx context.Context, fn func(context.Context, Info) error) error
+
+	// Close releases the internal resources and makes it so that a new
+	// snapshotter instance can be created using the same state directory.
```

Comment:
This seems like an implementation detail. Can't multiple snapshotter instance be made from a single state directory?